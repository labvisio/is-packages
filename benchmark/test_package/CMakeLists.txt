project(PackageTest CXX)
cmake_minimum_required(VERSION 2.8.12)

find_package(benchmark REQUIRED CONFIG)

add_executable(example example.cpp)
target_link_libraries(example benchmark::benchmark)
set_property(TARGET example PROPERTY CXX_STANDARD 11)
