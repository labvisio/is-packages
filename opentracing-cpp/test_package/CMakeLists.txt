project(PackageTest CXX)
cmake_minimum_required(VERSION 2.8.12)

find_package(opentracing-cpp CONFIG REQUIRED)

add_executable(example example.cpp)
target_link_libraries(example opentracing-cpp::opentracing-cpp)
set_property(TARGET example PROPERTY CXX_STANDARD 11)
