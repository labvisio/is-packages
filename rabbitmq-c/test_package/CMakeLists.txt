project(PackageTest CXX)
cmake_minimum_required(VERSION 2.8.12)

find_package(rabbitmq-c REQUIRED CONFIG)

add_executable(example example.cpp)
if(RABBITMQ_SHARED)
    target_link_libraries(example PRIVATE rabbitmq::rabbitmq)
else()
    target_link_libraries(example PRIVATE rabbitmq::rabbitmq-static)
endif()
