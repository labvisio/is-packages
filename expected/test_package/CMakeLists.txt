project(PackageTest CXX)
cmake_minimum_required(VERSION 2.8.12)

find_package(expected REQUIRED CONFIG)

add_executable(example example.cpp)
target_link_libraries(example expected::expected)
set_property(TARGET example PROPERTY CXX_STANDARD 11)
